http-netty/src/test/groovy/io/micronaut/http/netty/stream/HttpStreamsClientHandlerSpec.groovy

```diff
@@ -1,66 +0,0 @@
-package io.micronaut.http.netty.stream
-
-import io.netty.buffer.ByteBuf
-import io.netty.buffer.Unpooled
-import io.netty.channel.ChannelHandlerContext
-import io.netty.channel.ChannelOutboundHandlerAdapter
-import io.netty.channel.ChannelPromise
-import io.netty.channel.embedded.EmbeddedChannel
-import io.netty.handler.codec.http.DefaultHttpContent
-import io.netty.handler.codec.http.DefaultHttpRequest
-import io.netty.handler.codec.http.HttpContent
-import io.netty.handler.codec.http.HttpMethod
-import io.netty.handler.codec.http.HttpRequest
-import io.netty.handler.codec.http.HttpVersion
-import io.netty.handler.codec.http.LastHttpContent
-import reactor.core.publisher.Flux
-import spock.lang.Issue
-import spock.lang.Specification
-
-import java.nio.charset.StandardCharsets
-
-class HttpStreamsClientHandlerSpec extends Specification {
-    @Issue('https://github.com/micronaut-projects/micronaut-tracing/issues/316')
-    def 'out of order write'() {
```

Comment:
do we test this somewhere else?

Reply:
content isnt delayed anymore so the test case is obsolete